tests/node_8_5_0_ubuntu_build_test.yaml

```diff
@@ -0,0 +1,15 @@
+schemaVersion: '2.0.0'
+commandTests:
+- name: 'node version'
+  command: 'node'
+  args: ['--version']
+  expectedOutput: ['v8.5.0']
+metadataTest:
+  env:
+    - key : PORT
+      value: 8080
+    - key: NODE_ENV
+      value: production
+  exposedPorts: ["8080"]
+#  entrypoint: []
```

Comment:
remove

Reply:
I put this here a a stub as I believe we should test the entrypoint but I wasn't sure what it should be.  Will change to the appropriate value.

Reply:
going to drop this actually, ubuntu is the only image that has a defined entrypoint, the rest inherit it